fcd/pass_seseloop.cpp

```diff
@@ -195,11 +191,8 @@ namespace
 			auto constantI = ConstantInt::get(i32, i);
 			
 			SmallPtrSet<BasicBlock*, 4> updatedPredecessors;
-			auto useIter = thisBlock->use_begin();
-			while (useIter != thisBlock->use_end())
+			for (Use& blockUse : thisBlock->uses())
```

Comment:
Has this been verified to work? I'm pretty sure that I didn't use raw iterators for fun. Iterating on an iplist and deleting the current element usually crashes.